birdhouse/config/jupyterhub/jupyterhub_config.py.template

```diff
@@ -1,21 +1,49 @@
 import os
 import logging
+import subprocess
 
 c.JupyterHub.bind_url = 'http://:8000/jupyter'
 c.JupyterHub.hub_ip = 'jupyterhub'
 
+c.JupyterHub.authenticator_class = 'jupyterhub_magpie_authenticator.MagpieAuthenticator'
+c.MagpieAuthenticator.magpie_url = "http://magpie:2001"
```

Comment:
Nice, using internal docker network.  Do you know why historically we do not use that more often?  We tend to use the public ip `PAVICS_FQDN` which depends on external routing.  However, just to be safe, have to add a `links` section in `docker-compose.yml` of jupyterhub to magpie.

Reply:
I think they should be used more... is the `links` section necessary? The network created by docker-compose isn't enough?

Reply:
`links` is not necessary but might prevent problems later.  Ex: docker will start magpie before jupyterhub, if someone want to extract dependencies between the containers, `links` will explicitly show that dependency/connection.